inductiva/fluids/scenarios/wind_tunnel/README.md

```diff
@@ -0,0 +1,155 @@
+
+To initialize the scenario, the user can define the following parameters:
+- Dimensions of the wind tunnel in meters, e.g. `{"x": [-5, 15], "y": [-5, 5], "z": [0, 8]}`.
+- Airflow velocity in m/s, e.g. `[30, 0, 0]`. Notice that the airflow is injected from the negative x-direction.
+
+Now, the user is ready to simulate the steady state. Here, the user chooses the object to be inserted inside the wind tunnel. This object is defined with a geometry file in STL or OBJ format. Notice that, the required meshing step will automatically be made before starting the simulation. The meshing is done with the snappyHexMesh tool of OpenFOAM.
+
+The user is performing steady-state simulations of a vehicle inside the wind tunnel with air flowing. The simulation uses an iterative algorithm that converges to the steady-state of the flow. 
+The simulation parameters available for the user to configure are:
+- num_iterations: Set the maximum number of iterations for the algorithm to converge.
+- resolution: Controls the resolution of the meshing that is done prior to the simulation. The higher the resolution, the finer the meshing. Possibilities: "high", "medium", "low".
+
+Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`, `n_cores`.
+Launching a simulation returns a task object, which can be used to verify the status of the simulation, get the simulation outputs and access post-processing tools. See more in the [Tasks section](../../../../README.md).
+
+### Example:
+
+```python
+import inductiva
+
+# Initialize the scenario
+scenario = inductiva.fluids.scenarios.WindTunnel(
+    flow_velocity=[30, 0, 0],
+    dimensions={"x": [-5, 15], "y": [-5, 5], "z": [0, 8]})
+
+# Run a simulation
+task = scenario.simulate(
+    object_path="f1.obj",
+    num_iterations=1000, resolution=0.5,
+    run_async=True, n_cores=4)
+
+# Download the simulation output to your local machine.
+output = task.get_output()
+```
+
+This last step is essential to download the simulation files into your local machine 
+and apply the post-processing and visualizations to understand the aerodynamics of the object.
+
+## Output and Post-Processing
+
+The simulation output is comprised of several files spread throughout several directories. These files contain data about the airflow over the object, from which valuable metrics can be extracted to understand the aerodynamics of the object. To obtain these metrics a few post-processing steps are required to be done.
```

Comment:
several .... several....

Reply:
Yeah and try to keep it simple. Perhaps this paragraph can be replaced by just this:

"The output will contain several sub-directories with files that can be post-processed to extract relevant metrics, such as:"